doc/nitrocli.1

```diff
@@ -131,6 +131,15 @@ open.
 .TP
 \fBnitrocli hidden close
 Close a hidden volume.
+.TP
+\fBnitrocli fill\fR
```

Comment:
Should we mention that this command takes so long? I mean...1h is a different ballpark than anything I expected.

Reply:
Good point.